main/ertp/api/mint.md

```diff
@@ -1,31 +1,48 @@
 # Mint
 
-Only the mint can issue new digital assets, so only the holder of the mint can
-create new digital assets. These assets all have the same kind, which is called a
-[`Brand`](./brand.md).
+Only a `mint` can issue new digital assets, so only the holder of a `mint` can
+do so. A `mint` has a one-to-one relationship with both an `issuer` and a `brand`,
+and can only mint new assets of that `brand` and is the only `'mint` that can mint
+new `brand` type assets for its associated `brand`.
+
+## makeIssuerKit(allegedName, amountMathKind)
+- `allegedName` `{String}`
+- `amountMathKind` `{MathKind}`
+- Returns: `{Purse}`
+
+While not a method called on a `mint`, clearly it's important to know how to create a new `mint`. 
+`makeIssuerKit()` returns a new `issuer`, `mint`, `amountMath`, and `brand`. 
+See [here](./issuer.md#makeissuerkit-allegedname-amountmathkind) for details.
 
 ## mint.getIssuer()
 - Returns: `{Issuer}`
 
-Get the Issuer for this mint.
+Get the `Issuer` associated with this `mint`. A `mint` is always in an unchangable
+one-to-one relationship with an `issuer` from their creation. 
 
 ```js
-const { issuer, mint } = makeIssuerKit('bucks');
-const mintIssuer = mint.getIssuer();
+const { issuer: quatloosIssuer, mint: quatloosMint } = makeIssuerKit('quatloos');
+const quatloosMintIssuer = quatloosMint.getIssuer();
 
 // returns true
-Object.is(issuer, mintIssuer);
+Object.is(issuer, quatloosMintIssuer);
```

Comment:
```suggestion
issuer === quatloosMintIssuer
```

triple-equals is more common and more useful. I had to look up `Object.is()` and learned that it's an alternative check that handles fewer cases correctly.  I'll never use it.